mars/services/subtask/worker/processor.py

```diff
@@ -206,6 +206,7 @@ async def _execute_graph(self, chunk_graph: ChunkGraph):
                     chunk,
                     self.subtask.subtask_id,
                 )
+                self._datastore[chunk.op] = chunk
```

Comment:
This might be related to the question I asked, it looks a bit tricky to do so.

Reply:
Added `get_current_chunk`/`set_current_chunk` API instead.